src/eos-updater-poll.c

```diff
@@ -469,6 +469,9 @@ metadata_fetch_internal (OstreeRepo     *repo,
 
       if (local_error != NULL)
         {
+          if (g_error_matches (local_error, G_IO_ERROR, G_IO_ERROR_CANCELLED))
+            goto out;
```

Comment:
Nitpick: Rather than an `out` label here, I think I’d prefer to put the `g_propagate_error(); return FALSE` inline here. I prefer to avoid labels if possible, to make the control flow simpler.